docs/source/miniconda.rst

```diff
@@ -1,42 +1,58 @@
-
- `Installation
- instructions <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`__
-
- Other resources:
+Miniconda is a free minimal installer for conda. It is a small, bootstrap
+version of Anaconda that includes only conda, Python, the packages they depend
+on, and a small number of other useful packages, including pip, zlib and a
+few others. Use the ``conda install command`` to install 720+ additional conda
+packages from the Anaconda repository.
+
+`See if Miniconda is right for you <https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html#anaconda-or-miniconda>`_.
+
+Windows installers
+==================
+
+.. csv-table:: Windows
+   :header: Python version,Name,Size,SHA256 hash
+   :widths: 5, 10, 5, 80
+
+   Python 3.7,`Miniconda3 Windows 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe>`_,76.2 MiB,``a3a8921c2dec37f4ef37b9fa7b337dba237ccacec56bed3d8b8c300ed852c84f``
+   ,`Miniconda3 Windows 32-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86.exe>`_,70.7 MiB,``789a0cafbc4c43fb53facced1a32203865bc1600e5baf70e97e0ce3d64aebd4b``
+   Python 2.7,`Miniconda2 Windows 64-bit <https://repo.anaconda.com/miniconda/Miniconda2-latest-Windows-x86_64.exe>`_,75.2 MiB,``9cf92cb336fd29c4fabbf22523d71a52623bf5ed7895d6cd079d569af5e4b7cd``
+   ,`Miniconda2 Windows 32-bit <https://repo.anaconda.com/miniconda/Miniconda2-latest-Windows-x86.exe>`_,69.5 MiB,``a90d5b689f8a57c0da85ad77d3efa683a23da9ddb19429587635d222d5d1005c``
+
+
+MacOSX installers
+=================
+
+.. csv-table:: MacOSX
+   :header: Python version,Name,Size,SHA256 hash
+   :widths: 5, 10, 5, 80
+
+   Python 3.7,`Miniconda3 MacOSX 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh>`_,54.6 MiB,``c8b31ea37b0b6a3e2fb19990ef895ab5cf1c095f8e9138defac95ee88e70920d``
+   ,`Miniconda3 MacOSX 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg>`_,84.4 MiB,``e51804f0a55b1aac2200bbe21f06fe519536071ec14c8cb6d29f1ae7ec5dbfaf``
+   Python 2.7,`Miniconda2 MacOSX 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda2-latest-MacOSX-x86_64.sh>`_,44.3 MiB,``9e73501268c2a288fdb0f3ddee01f1162a29dc2671f63b659ae447d61da08810``
+   ,`Miniconda2 MacOSX 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda2-latest-MacOSX-x86_64.pkg>`_,59.1 MiB,``97de47ce5028d382d436997911138db2fa473644de549dc6d888bbc2f41a1a8f``
+
+Linux installers
+================
+
+.. csv-table:: Linux
+   :header: Python version,Name,Size,SHA256 hash
+   :widths: 5, 10, 5, 80
+
+   Python 3.7,`Miniconda Linux 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh>`_,84.5 MiB,``8a324adcc9eaf1c09e22a992bb6234d91a94146840ee6b11c114ecadafc68121``
+   ,`Miniconda3 Linux 32-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86.sh>`_,66.3 MiB,``f387eded3fa4ddc3104b7775e62d59065b30205c2758a8b86b4c27144adafcc4``
```

Comment:
This size doesn't match what I'm seeing, even though the hash matches, so I'm guessing this was human error (either mine or yours 😄).  What I see is 62.7MiB or 65.7MB. (65741329 bytes)